tools/aa/commands/InitCommand.java

```diff
@@ -25,13 +25,12 @@
 import java.nio.file.Paths;
 import org.eclipse.jgit.api.Git;
 
-/** A command to init a base folder.
+/**
+ * A command to init a base folder.
  *
- * Usage:
- * bazel run //tools/aa:aa_tool -- init --base_path </path/to/base/folder>
- * or, if aa is already set up:
- * aa init --base_path </path/to/base/folder>
-*/
+ * <p>Usage: bazel run //tools/aa:aa_tool -- init --base_path </path/to/base/folder> or, if aa is
```

Comment:
Please make this a block comment and not Javadoc, so we can conserve the formatting:
```
/**
 * Javadoc
 */
```
```
/*
 * Block comment
 */
```

Reply:
@oferb ok. do you think we can automate this check somehow? we still *do* need `Javadoc` sometimes, right?